scripts/perftest/genMultinomialData.sh

```diff
@@ -33,31 +33,31 @@ SPARSE_SP=0.01
 export HADOOP_CLIENT_OPTS="-Xmx2048m -Xms2048m -Xmn256m"
 
 #generate XS scenarios (80MB)
-${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-args 10000 1000 $DENSE_SP 5 0 $BASE/X10k_1k_dense_k5 $BASE/y10k_1k_dense_k5 $FORMAT 1
-${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-args 10000 1000 $SPARSE_SP 5 0 $BASE/X10k_1k_sparse_k5 $BASE/y10k_1k_sparse_k5 $FORMAT 1
+${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-nvargs nr=10000 nf=1000 sparse=$DENSE_SP numcat=5 icpt=0 X=$BASE/X10k_1k_dense_k5 Y=$BASE/y10k_1k_dense_k5 fmt=$FORMAT
+${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-nvargs nr=10000 nf=1000 sparse=$SPARSE_SP numcat=5 icpt=0 X=$BASE/X10k_1k_sparse_k5 Y=$BASE/y10k_1k_sparse_k5 fmt=$FORMAT
 ${CMD} -f extractTestData.dml $DASH-args $BASE/X10k_1k_dense_k5 $BASE/y10k_1k_dense_k5 $BASE/X10k_1k_dense_k5_test $BASE/y10k_1k_dense_k5_test $FORMAT
 ${CMD} -f extractTestData.dml $DASH-args $BASE/X10k_1k_sparse_k5 $BASE/y10k_1k_sparse_k5 $BASE/X10k_1k_sparse_k5_test $BASE/y10k_1k_sparse_k5_test $FORMAT
 
 ##generate S scenarios (80MB)
-#${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-args 100000 1000 $DENSE_SP 5 0 $BASE/X100k_1k_dense_k5 $BASE/y100k_1k_dense_k5 $FORMAT 1
-#${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-args 100000 1000 $SPARSE_SP 5 0 $BASE/X100k_1k_sparse_k5 $BASE/y100k_1k_sparse_k5 $FORMAT 1
+#${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-nvargs nr=100000 nf=1000 sparse=$DENSE_SP numcat=5 icpt=0 X=$BASE/X100k_1k_dense_k5 Y=$BASE/y100k_1k_dense_k5 fmt=$FORMAT
+#${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-nvargs nr=100000 nf=1000 sparse=$SPARSE_SP numcat=5 icpt=0 X=$BASE/X100k_1k_sparse_k5 Y=$BASE/y100k_1k_sparse_k5 fmt=$FORMAT
 #${CMD} -f extractTestData.dml $DASH-args $BASE/X100k_1k_dense_k5 $BASE/y100k_1k_dense_k5 $BASE/X100k_1k_dense_k5_test $BASE/y100k_1k_dense_k5_test $FORMAT
 #${CMD} -f extractTestData.dml $DASH-args $BASE/X100k_1k_sparse_k5 $BASE/y100k_1k_sparse_k5 $BASE/X100k_1k_sparse_k5_test $BASE/y100k_1k_sparse_k5_test $FORMAT
 #
 ##generate M scenarios (8GB)
-#${CMD} -f ../datagen/genRandData4Multinomial.dml $DASH-args 1000000 1000 $DENSE_SP 5 0 $BASE/X1M_1k_dense_k5 $BASE/y1M_1k_dense_k5 $FORMAT 1
```

Comment:
@asurve could you please clarify why the value `1` is required at the end?. I have removed it as its not required in  `genRandData4Multinomial `.